### PR TITLE
Fix Tool to Allow Partial Activation of Launch Plans

### PR DESCRIPTION
**Pull request by matthewphsmith**

### CODE REVIEW

File: flytekit/tools/module_loader.py [hunk 1]
```diff
@@ -90,7 +95,8 @@ def iterate_registerable_entities_in_order(pkgs, ignore_entities=None, include_e
         entities will be taken.  Only one of ignore_entities or include_entities can be set.
     :param set[type] include_entities: If specified, include these entities while doing a topological sort.  All
         other entities will be ignored.  Only one of ignore_entities or include_entities can be set.
-    :param bool only_include_defs: If false, we will possibly return duplicate references to an entity.
```
Comment (chanadian) [inline flytekit/tools/module_loader.py]: why is this removed too?

Comment (matthewphsmith) [reply flytekit/tools/module_loader.py]: I changed the name and doc string
